packages/react-components/react-tabs/library/src/components/Tab/useTabStyles.styles.ts

```diff
@@ -440,27 +447,28 @@ const useContentStyles = makeStyles({
 export const useTabStyles_unstable = (state: TabState): TabState => {
   'use no memo';
 
+  useTabIndicatorStyles_unstable(state);
+
+  useTabButtonStyles_unstable(state, state.root);
+
+  useTabContentStyles_unstable(state);
+
+  return state;
+};
+
+/** Apply styling to the Tab indicator slot based on the state */
```

Comment:
can we add more verbose docs about these new low level style hook apis ?

something like how/why would one use these including mentioning that these are part off/called  from main  `useTabStyles_unstable` so for composition this should be used only in very rare cases etc ?